src/alto/app/estimator.py

```diff
@@ -142,18 +104,18 @@ def do_request_from_str(input_str, alto_server):
 	Returns:
 		A JSON string representing the throughput for each flow
 	"""
-    import argparse
-    import sys
+	import argparse

```

Comment:
Although we didn't add any lintrc to enforce the coding style, could you change all the indents to 4 space soft tabs? Thanks.